product_state/readme/ROADMAP.rst

```diff
@@ -0,0 +1 @@
+File data/product_state_data.xml will be moved to demo/product_state_demo.xml since version 15.0
```

Comment:
Please remove this line. The file was indeed moved in the 15.0 migration but it was reverted in https://github.com/OCA/product-attribute/commit/6079fdb26a6b792ff1c26966348c3f08441debf4.